setup.py

```diff
@@ -51,7 +51,7 @@ def get_version(filename):
         'jsonschema~=3.0',
         'pathlib2>=2.3.0;python_version<="3.4"',
         'importlib_resources~=1.0.2;python_version=="3.4"',
-        'importlib_resources~=1.4;python_version<"3.7" and python_version!="3.4"',
+        'importlib_resources>=1.4,<4;python_version<"3.7" and python_version!="3.4"',
```

Comment:
Does version 4 have breaking changes?

Reply:
Version 4 hasn't been released yet, 3.x is the current latest -- this is just being safe as we don't know what the breaking change will be when it comes.